### PR TITLE
Updated stock requisition completion to be possible within approval p…

### PR DESCRIPTION
**Pull request by patryllus**
https://openmrs.atlassian.net/browse/O3-3369
[Screencast from 05-06-2024 04:21:37 ALASIRI.webm](https://github.com/openmrs/openmrs-esm-stock-management/assets/1901372/77f8c3f4-39e3-407e-b8b2-4845a3fd67d4)

### CODE REVIEW

**Comment by makombe** [comment]
@patryllus please ticket this [here](https://openmrs.atlassian.net/browse/O3-2770) for tracking purposes.

**Comment by patryllus** [comment]
@makombe Here : https://openmrs.atlassian.net/browse/O3-3369

File: src/stock-operations/add-stock-operation/stock-operation-submission.component.tsx [hunk 1]
```diff
@@ -111,10 +111,15 @@ const StockOperationSubmission: React.FC<StockOperationSubmissionProps> = ({
                   kind="primary"
                   onClick={async () => {
                     delete model?.dateCreated;
-                    model.status = "COMPLETED";
+                    delete model?.status;
```
Comment (jabahum) [inline src/stock-operations/add-stock-operation/stock-operation-submission.component.tsx:114]: @patryllus what happens when you click complete button

**Review by jabahum — COMMENTED** [review]
@patryllus what happens to the completed button

**Comment by patryllus** [comment]
> @patryllus what happens to the completed button
Screenshot from 2024-06-05 20-36-43

@jabahum it pops up the modal to requesting 'Would you really like to Complete the operation ?' . If the user clicks submit, the operation is completed.

**Review by jabahum — APPROVED** [review]
LGTM

**Pull request merged**